f8a_worker/workers/CVEchecker.py

```diff
@@ -127,6 +146,19 @@ def _query_ossindex(self, arguments):
     def _npm_scan(self, arguments):
         return self._query_ossindex(arguments)
 
+    @staticmethod
+    def update_depcheck_db_on_s3():
+        """Update OWASP Dependency-check DB on S3"""
+        s3 = StoragePool.get_connected_storage('S3VulnDB')
+        depcheck = os.path.join(configuration.OWASP_DEP_CHECK_PATH, 'bin',
+                                'dependency-check.sh')
+        with tempdir() as temp_data_dir:
+            s3.retrieve_depcheck_db_if_exists(temp_data_dir)
+            # give DependencyCheck 30 minutes to download the DB
+            TimedCommand.get_command_output([depcheck, '--updateonly', '--data', temp_data_dir],
+                                            timeout=1800)
```

Comment:
Do we need 30 minutes here? I'm afraid we can miss visibility timeout that is [30 mins currently](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/celery_settings.py#L56).